crate/operator/change_plan.py

```diff
@@ -0,0 +1,121 @@
+import logging
+from typing import Any
+
+import kopf
+from kubernetes_asyncio.client import AppsV1Api
+from kubernetes_asyncio.client.api_client import ApiClient
+from webhooks import WebhookChangePlanPayload, WebhookEvent, WebhookStatus
```

Comment:
```suggestion
from crate.operator.webhooks import WebhookChangePlanPayload, WebhookEvent, WebhookStatus
```

This is what's causing the docs build to fail.